wordpress-mobile/WordPress-iOS
# Reader: Add blogs and tags empty filter sheet view

Part of #22403

## Description

Adds the empty views for when the user has no blog or tag subscriptions.

## Screenshots

| Light | Dark |
|---|---|
| ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-01-16 at 18 20 10](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/71493993-0e8e-44ac-85cd-649e28cea146) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-01-16 at 18 20 18](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/18e7833e-8892-4af1-bbc8-2992d9ddacd7) |
| ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-01-16 at 18 20 29](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/5991c8ae-2513-4cdc-a014-e7be220276b6) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-01-16 at 18 20 35](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/8ea450d7-194b-40d3-9b01-d0e1a96d14a9) |

## Testing

To test:
- Set up an account with no blog/tag subscriptions
- Launch Jetpack and login
- Navigate to the `Subscriptions` Reader feed
- Tap on `0 Blogs` pill button
- 🔎 **Verify** the bottom sheet appears with the new UI for no blog subscriptions
- Tap on `Search for blog`
- 🔎 **Verify** the blog search screen appears
- Dismiss the screens until back at the feed
- Tap on `0 Tags`
- 🔎 **Verify** the bottom sheet appears with the new UI for no tag subscriptions
- Tap on `Suggested tags`
- 🔎 **Verify** the tags cloud picker appears
- Dismiss the picker
- Tap on `Search for a tag`
- 🔎 **Verify** the manage screen appears with the `Followed Topics` tab selected

## Regression Notes
1. Potential unintended areas of impact
Reader filter sheet

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A
PR submission checklist:


UI Changes testing checklist:
